build.gradle

```diff
@@ -0,0 +1,152 @@
+allprojects {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    apply plugin: "java"
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
+
+    apply plugin: ProvidedPlugin
+    apply plugin: "maven-publish"
+
+    task sourcesJar(type: Jar, dependsOn: classes) {
+        classifier 'sources'
+        from sourceSets.main.allJava
+    }
+
+    // not working yet:
```

Comment:
Yep. Not sure why javadoc publishing isn't working, something about the classpath I suspect, but I doubt anyone will miss javadocs enough to care, ya?